kubernetes/annotations_and_labels.md

```diff
@@ -16,9 +16,10 @@ This page defines common annotations and labels we set in Kubernetes objects.
 ## Common Labels
 
 - `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This together with `version`
-  label should also be used as replicas selector. E.g. `app=kvm-operator,version=1.0.0`. Exceptions can
-  be made to accomodate for adherence to existing selectors upstream.
+  to every Kubernetes resource associated with an application. E.g.
+  `app=kvm-operator`. This is softly deprecated and should be replaced with
+  `app.kubernetes.io/name`, it's only kept to avoid breaking existing
+  workflows.
```

Comment:
I'd put it like:

```
- `app` - should be set to the same value as `app.kubernetes.io/name`. It's only kept
  to avoid breaking existing workflows.
```

Reply:
Done